src/multisig/multisigTransactionsFactory.ts

```diff
@@ -230,6 +216,21 @@ export class MultisigTransactionsFactory extends SmartContractTransactionsFactor
         }).build();
     }
 
+    private mapTokenPayment(options: resources.ProposeTransferExecuteEsdtInput) {
```

Comment:
Can be plural, since there are more payments.

Also, we should specify the return type.